ExilenceNextApp/src/components/item-table/ItemTableContainer.tsx

```diff
@@ -333,7 +348,18 @@ const ItemTableContainer = ({
           )}
         </Grid>
       </Box>
-      <TableWrapper instance={instance} setInitialState={setInitialState} />
+      <TableWrapper
+        instance={instance}
+        setInitialState={setInitialState}
+        noRowsMessage={
+          uiStateStore.netWorthSessionOpen &&
+          activeProfile?.session.snapshotPreviewIndex &&
+          activeProfile?.session.snapshotPreviewIndex >= 100
+            ? 'label.item_table_placeholder_net_worth_session_preview'
+            : undefined
```

Comment:
```suggestion
          uiStateStore.netWorthSessionOpen &&
          activeProfile?.session?.snapshotPreviewIndex >= 100
            ? 'label.item_table_placeholder_net_worth_session_preview'
            : undefined
```
Can you create separate variable for this logic and use it here?

Reply:
```suggestion
          uiStateStore.netWorthSessionOpen &&
          (activeProfile?.session?.snapshotPreviewIndex || 0) >= 100
            ? 'label.item_table_placeholder_net_worth_session_preview'
            : undefined
```

Here a better approch

Reply:
And yes I should use a variable too.

Reply:
Yes, I'll add a variable